spec/ingram_micro/elements/sales_order_header_spec.rb

```diff
@@ -6,6 +6,7 @@
 
     context 'when using the domestic schema' do
       it "returns the correct default values hash" do
+        IngramMicro.configuration.international = false
```

Comment:
don't forget to add to readme why there are 2 schemas and what's different between the two? you can do it after the other changes, since we're pressed for time to get this pr out today